game.go

```diff
@@ -14,26 +14,52 @@ const (
 type Game struct {
     board [][]uint8
     currentMove Player
+    gameBoardSize int //added board size variable for testing out of scope moves
 }
 
 func NewGame(boardSize int) Game {
     board := make([][]uint8, boardSize)
     for i := range board {
         board[i] = make([]uint8, boardSize)
     }
-    newGame := Game{board: board, currentMove: DARK}
+    board = boardStars(board, boardSize)
+    newGame := Game{board: board, currentMove: DARK, gameBoardSize: boardSize}
     return newGame
 }
 
+func boardStars(board [][] uint8, boardSize int) [][] uint8 {
+    center := boardSize/2
```

Comment:
nit:
would `boardSize / 2` look better?

Reply:
Yeah, you're right. done